web/src/components/netflow-traffic.tsx

```diff
@@ -453,10 +453,24 @@ export const NetflowTraffic: React.FC<NetflowTrafficProps> = ({
             model.setStats(stats);
           })
           .catch(err => {
-            model.setFlows([]);
-            model.setMetrics(defaultNetflowMetrics);
-            model.setError(getHTTPErrorDetails(err, true));
-            model.setWarning(undefined);
+            const errStr = getHTTPErrorDetails(err, true);
+            if (isPromMissingLabelError(errStr)) {
+              model.filters.list.forEach((filter) => {
+                const fieldName = model.config.fields.find((field) => field.filter === filter.def.id)?.name;
+                if (!fieldName) {
+                  console.error("can't find field:", filter.def.id);
+                } else if (errStr.includes(fieldName)) {
+                  model.disabledFilters[fieldName] = filter.values.map(fv => fv.v).join(',');
+                }
+              });
+              model.setDisabledFilters({ ...model.disabledFilters });
+              tick();
```

Comment:
Need to test how this will react :thinking: 

Also we need to warn the user with a popup but it may be strange if the user actually just added the filter messing up with the query